src/postprocess/chi0.jl

```diff
@@ -134,6 +134,7 @@ function apply_χ0(ham, δV, ψ, occupation, εF, eigenvalues; droptol=0,
 
             # Sternheimer contributions. TODO add preconditioning here
             !(sternheimer_contribution) && continue
+            fnk = filled_occ * Smearing.occupation(model.smearing, (εmk-εF) / temperature)
```

Comment:
Careful line below

Reply:
Just remove occupation arg?

Reply:
Here and in the other chi0 probably

Reply:
Not sure what you mean by the "other" chi0.

Reply:
I meant compute_chi0 but I see that's irrelevant (although we could have a custom temperature there as well?)